addons/binding/org.openhab.binding.allplay/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,177 @@
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="currentalbum">
+		<item-type>String</item-type>
+		<label>Current Album</label>
+		<description>Album of the track currently playing</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="currentartist">
+		<item-type>String</item-type>
+		<label>Current Artist</label>
+		<description>Artist of the track currently playing</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="currentduration" advanced="true">
+		<item-type>Number</item-type>
+		<label>Current Duration</label>
+		<description>Duration in milliseconds of the track currently playing</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="currentgenre" advanced="true">
+		<item-type>String</item-type>
+		<label>Current Genre</label>
+		<description>Genre of the track currently playing</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="currenttitle">
+		<item-type>String</item-type>
+		<label>Current Title</label>
+		<description>Title of the track currently playing</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="currentuserdata" advanced="true">
+		<item-type>String</item-type>
+		<label>Current User Data</label>
+		<description>Custom user data (e.g. name of radio station) of the track currently playing</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="currenturl" advanced="true">
+		<item-type>String</item-type>
+		<label>Current URL</label>
+		<description>URL of the track or radio station currently playing</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="loopmode" advanced="true">
+		<item-type>String</item-type>
+		<label>Loop Mode</label>
+		<description>Loop mode of the speaker (ONE, ALL, NONE)</description>
+		<state>
+			<options>
+				<option value="NONE">No Repeat</option>
+				<option value="ONE">Repeat Track</option>
+				<option value="ALL">Repeat Playlist</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="mute" advanced="true">
+		<item-type>Switch</item-type>
+		<label>Mute</label>
+		<description>Set or get the mute state of the speaker</description>
+	</channel-type>
+	<channel-type id="playstate">
+		<item-type>String</item-type>
+		<label>State</label>
+		<description>The State channel contains state of the Speaker, e.g. PLAYING, STOPPED,...</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="shufflemode" advanced="true">
+		<item-type>String</item-type>
+		<label>Shuffle Mode</label>
+		<description>Shuffle mode of the speaker (LINEAR, SHUFFLE)</description>
+		<state>
+			<options>
+				<option value="LINEAR">Linear</option>
+				<option value="SHUFFLE">Shuffle</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="stop">
```

Comment:
As far as I can see, the control channel only pauses the speaker. The difference for the speaker is probably that with pause, the current position is retained, while with stop it is reset to the start of the track (playlist?). Also, at least for my speakers, they will not go into standby after some minutes if I use pause, while with stop they do.